data/mahesh.json

```
@@ -0,0 +1,45 @@
+{
+  "name": "Mahesh Kasbe",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "**Open Source** Advocate | DevOps ",
+  "tags": ["Open Source", "DevOps", "Kubernetes", "Docker", "Node.js"],
+  "socials": [
+    { "platform": "twitter", "url": "https://twitter.com/maheshstwt" },
+    { "platform": "github", "url": "https://github.com/Mahesh-Kasabe" },
+    { "platform": "website", "url": "https://mahesh-kasabe.github.io/" }
+  ],
+  "avatar": "https://avatars.githubusercontent.com/u/60398112?s=400&u=f0abce668aeb1611b03f279d9596c5674ab757c6&v=4",
```

You can use this link instead to have it shorter and updated automatically. 🙂
```suggestion
  "avatar": "https://github.com/Mahesh-Kasabe.png",
```